api/v3/commonservice_types.go

```diff
@@ -64,7 +64,7 @@ type Bedrockshim struct {
 type BedrockOperator struct {
 	Name               string `json:"name,omitempty"`
 	Version            string `json:"version,omitempty"`
-	Status             string `json:"status,omitempty"`
+	OperatorStatus     string `json:"operatorStatus,omitempty"`
```

Comment:
Will this Change in this field cause an upgrade issue?

Reply:
it should be fine. I tested the upgrade path 3.11 -> my dev build:
1. fresh installed 3.11 from quay.io
2. replace installed catalogsource with my catalogsource includes this change
3. run `oc apply -f bundle/manifests/operator.ibm.com_commonservices.yaml` on testing cluster

Reply:
It looks good from the code side. The new operator slice will totally overwrite the previously status. So it would be fine during the upgrade.